repos/fdbt-site/src/utils/apiUtils/matching.ts

```diff
@@ -169,6 +169,7 @@ export const getMatchingProps = async (
     matchingAttribute: MatchingWithErrors | object | undefined,
 ): Promise<{ props: MatchingProps }> => {
     const serviceAttribute = getSessionAttribute(ctx.req, SERVICE_ATTRIBUTE);
+
```

Comment:
This function is being called on the Outbound matching page, and the inbound matching page. It is currently using the same service for both pages, and not using the 2nd service the user selected. We need to be rendering the inbound matching page using the 2nd service they select.

Reply:
solved